x-pack/test/functional/apps/machine_learning/data_frame_analytics/regression_creation.ts

```diff
@@ -70,6 +71,11 @@ export default function({ getService }: FtrProviderContext) {
           await ml.dataFrameAnalyticsCreation.setJobId(testData.jobId);
         });
 
+        it('inputs the job description', async () => {
+          await ml.dataFrameAnalyticsCreation.assertJobDescriptionInputExists();
+          await ml.dataFrameAnalyticsCreation.setJobDescription(testData.jobDescription);
+        });
+
```

Comment:
Similar to outlier detection: we should assert the job description in the table after job creation.

Reply:
Added in 6116a586782001a074d9582df01434f18a3c8975